cachito/web/api_v1.py

```diff
@@ -716,3 +718,87 @@ def get_content_manifest_by_requests():
     if len(requests) > 1:
         deep_sort_icm(assembled_icm)
     return send_content_manifest_back(assembled_icm)
+
+
+class RequestMetricsArgs(pydantic.BaseModel):
+    """Query parameters for /request-metrics endpoint."""
+
+    start_date: Optional[date] = None
+    end_date: Optional[date] = None
```

Comment:
Could the naming and behavior be consistent with #570?

Reply:
yes please, make naming consistent